docker/Dockerfile.centos

```diff
@@ -45,6 +45,13 @@ RUN wget -q http://linuxsoft.cern.ch/cern/scl/slc6-scl.repo && mv slc6-scl.repo
 RUN yum install -y devtoolset-3-gcc-c++
 RUN echo 'source /opt/rh/devtoolset-3/enable' >> ~/.bashrc
 
+# This is workaround to be able to compile boringssl with -DOPENSSL_C11_ATOMIC as while we use a recent gcc installation it still needs some
+# help to define static_assert(...) as otherwise the compilation will fail due the system installed assert.h which missed this definition. 
+run mkdir ~/.include
+run echo '#include "/usr/include/assert.h"' >>  ~/.include/assert.h
+run echo '#define static_assert _Static_assert'  >>  ~/.include/assert.h
+run echo 'export C_INCLUDE_PATH="$HOME/.include/"' >>  ~/.bashrc
```

Comment:
Huh. I'm a little surprised you need this given the atomics path depends on `stdatomic.h` (also a C11 header). But I dunno, old toolchains are weird I guess. :-(

By the way, CentOS 6 is nine years old, stopped getting "full updates" three years ago and will stop getting "maintenance updates" altogether in three months.

Reply:
Yeah its weird... that said it should be "ok" :)

Reply:
its because we use centos6 for building...